conduct.md

```diff
@@ -133,44 +148,49 @@ the project after its launch. Through postmortems we can look at what was
 successful and what failed and needs improvement. These discussions also
 provide great insight in how to improve in future projects.
 
-* Acceptable Conduct (A):
-  * Members shall
-    * Work as a team to create a postmortem plan to review and fix sections of the project.
-    * Participate in the discussions in a respectful and courteous manner
-    while adhering to the communications section of the code of conduct.
-    * Provide constructive feedback and propose solutions for issues faced.
-    * Receive feedback and discuss how it could be used to improve future projects and assignments.
-    * :heavy_exclamation_mark: TODO: add any additional items in the A category for the postmortems.
-
-* Unacceptable Conduct (U):
-  * Members shall not
-    * Insult others while presenting criticism.
-    * Blame other members for any issues with the final product.
-    * Refuse to collaborate in fixing remaining issues while a project or lab has not been completed.
-    * :heavy_exclamation_mark: TODO: add any additional items in the U category for the postmortems.
+- Acceptable Conduct (A):
+
+  - Members shall
+
+    - Work as a team to create a postmortem plan to review and fix sections of the project.
+    - Participate in the discussions in a respectful and courteous manner
+      while adhering to the communications section of the code of conduct.
+    - Provide constructive feedback and propose solutions for issues faced.
+    - Receive feedback and discuss how it could be used to improve future projects and assignments.
+
+    - Consider all possible angles and opions when determining what is good feedback.
+    - Thoroughly consider all feedback regardless of how it may initially seem.
```

Comment:
Please let the group responsible for postmortems know about these changes. Additionally, you should keep the `TODO` at the bottom of the list until the group responsible finalizes their contribution